lib/error-res.js

```diff
@@ -1,48 +1,80 @@
 'use strict';
 
+/**
+ * Generates error handlers
+ */
+const handlerGenerator = (statusCode, statusMessage) =>
+  (res, msg) => {
+    res.statusCode = statusCode;
+    res.statusMessage = statusMessage;
+    res.write(JSON.stringify({ error: msg ? msg : `${statusCode} ${statusMessage}` }));
+  };
+
 /**
 * Ends request with a 401
 * @param {Response} res
 */
-const endUnauthorised = res => {
-  res.statusCode = 401;
-  res.statusMessage = 'Unauthorised';
-  res.write(JSON.stringify({ error: '401 Unauthorised' }));
-  res.end();
-};
+const endUnauthorised = handlerGenerator(
```

Comment:
Even in single line `const endUnauthorised = handlerGenerator(401, 'Unauthorised');`

Reply:
Single line tends to exceed 80 chars